### DISCUSSION TITLE
@stack/@push alternative

### DISCUSSION THREAD

**Discussion by gjm** [livewire/livewire#4288] [4. General]
Hi all.

The hability to push JS code to a stack and running it after a component refresh was a great feature. It was rolled back in 2.7.1 because it broke some users' usage.

Is there an alternative available? Somehow force this manually? 

Thanks.

**Comment by joshhanley** [comment]
@gjm not yet, but we haven't removed the feature, just turned it off for now. It's something we definitely want to get sorted!
